dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1412,14 +1414,48 @@ public boolean delete(Contentlet contentlet, User user,boolean respectFrontendRo
         return true;
     }
 
-    @WrapInTransaction
+
+
     @Override
-    public boolean deleteByHost(Host host, User user, boolean respectFrontendRoles)
+    public boolean deleteByHost(final Host host, final User user, final boolean respectFrontendRoles)
             throws DotDataException, DotSecurityException {
-        List<Contentlet> contentletsToDelete = findContentletsByHost(host, user,
-                respectFrontendRoles);
 
-        return deleteContentlets(contentletsToDelete, user, respectFrontendRoles, true);
+        
+        final DotConnect db = new DotConnect();
+
+        List<String> deleteMe = db.setSQL("select working_inode  from identifier, contentlet_version_info where identifier.id = contentlet_version_info.identifier and host_inode=? and asset_type='contentlet'")
+                .addParam(host.getIdentifier())
+                .setMaxRows(200)
+                .loadObjectResults()
+                .stream()
+                .map(map->(String)map.get("working_inode"))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "working_inode" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'map' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)